waspc/cli/exe/Main.hs

```diff
@@ -89,36 +95,40 @@ printUsage :: IO ()
 printUsage =
   putStrLn $
     unlines
+{- ORMOLU_DISABLE -}
```

Comment:
Why did you disable it?

Reply:
Because I wanted to format nicely the stuff below, usage strings, so you can read it same as you would read it in the terminal! Otherwise it was a bit of a mess.